ci-operator/config/openshift/installer/openshift-installer-master.yaml

```
@@ -227,7 +227,7 @@ tests:
     workflow: openshift-e2e-gcp
 - as: e2e-alibaba
   optional: true
-  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|terraform/providers)/(aws|azure|azurestack|baremetal|gcp|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
+  run_if_changed: (^(data/.*|pkg/.*)/alibabacloud/)|(^vendor/github.com/aliyun/)|(^terraform/providers/alicloud)
```

Could we simplify this some to just look for `alibabacloud`, `aliyun`, or `alicloud`? There is a slight chance of some false positives on the matching, but it makes it much easier to read and maintain.
```suggestion
  run_if_changed: (alibabacloud)|(aliyun)|(alicloud)
```